nsxt/resource_nsxt_node_user_test.go

```diff
@@ -181,14 +197,15 @@ resource "nsxt_node_user" "test" {
 }`, attrMap["active"], attrMap["full_name"], attrMap["password"], username, attrMap["password_change_frequency"], attrMap["password_change_warning"])
 }
 
-func testAccNodeUserUpdate(username string) string {
+func testAccNodeUserUpdate(username, password string) string {
```

Comment:
sorry for pedant comment, but can be just take `password` from `attrMap` without passing it as param here?

Reply:
@annakhm note that there are two update cases, with and without password change (as that requires some extra logic). If we use the password from attrMap we need an indication whether password should change or not.